include/deal.II/distributed/tria_base.h

```diff
@@ -132,6 +132,13 @@ namespace parallel
      */
     types::subdomain_id locally_owned_subdomain () const;
 
+    /**
+     * Returns a list of MPI ranks of the processors that have at least one
+     * ghost cell adjacent to our cells. Note that if @p i is contained in the
+     * list of processor @p j, then @p j will also be contained in the list of
+     * processor @p i.
+     **/
+    virtual const std::vector<unsigned int> &ghost_neighbors () const;
```

Comment:
i wonder if `ghost_owners` is a more descriptive name? I think that what you do: if cell is ghost from the perspective of the current processor, put it's subdomain id (== mpi rank) to the list.


> ...at least one ghost cell adjacent to our cells

I may be wrong, but i read that line as if these cells are ghosts for other processors,
whereas in reality these ghost cells are ghost only for the current processor. 
Perhaps the description could be slightly changed to reflect it.